### PR TITLE
DTRUNEONE-4583 Reverting package changes and updating eslint base dep…

### PR DESCRIPTION
**Pull request by skoong**
Changes in this Pull Request:

- Reverting version and invalid package-lock.json
- Upgrading eslint-config-airbnb-base to match the eslint version dev dependency and applying formatting changes
- Adding return to terminate after promise rejection in Encryption.js

**Tests**
```
  438 passing (3s)
  2 pending
```

### CODE REVIEW

**Review by hlahlou-pp-dev — APPROVED** [review]

File: test/Hyperwallet.spec.js [hunk 1]
```diff
@@ -2303,7 +2355,9 @@ describe("Hyperwallet", () => {
         /** @test {Hyperwallet#listPrepaidCards} */
         it("should do get call with options", () => {
             const callback = () => null;
-            const options = { status: "test-status", createdBefore: "test-createdBefore", createdAfter: "test-createdBefore", sortBy: "test-sortBy", offset: "test-offset", limit: "test-limit" };
+            const options = {
+                status: "test-status", createdBefore: "test-createdBefore", createdAfter: "test-createdBefore", sortBy: "test-sortBy", offset: "test-offset", limit: "test-limit",
```
Comment (wmews-hw) [inline test/Hyperwallet.spec.js:2359]: shouldn't it be formatted in many lines?

File: test/Hyperwallet.spec.js [hunk 2]
```diff
@@ -2993,7 +3066,9 @@ describe("Hyperwallet", () => {
         /** @test {Hyperwallet#listBalancesForUser} */
         it("should do get call with options", () => {
             const callback = () => null;
-            const options = { currency: "test-currency", sortBy: "test-sortBy", offset: "test-offset", limit: "test-limit" };
+            const options = {
+                currency: "test-currency", sortBy: "test-sortBy", offset: "test-offset", limit: "test-limit",
```
Comment (wmews-hw) [inline test/Hyperwallet.spec.js:3070]: shouldn't it be formatted in many lines same as the other?

File: test/Hyperwallet.spec.js [hunk 3]
```diff
@@ -3067,7 +3142,9 @@ describe("Hyperwallet", () => {
         /** @test {Hyperwallet#listBalancesForPrepaidCard} */
         it("should do get call with options", () => {
             const callback = () => null;
-            const options = { currency: "test-currency", sortBy: "test-sortBy", offset: "test-offset", limit: "test-limit" };
+            const options = {
+                currency: "test-currency", sortBy: "test-sortBy", offset: "test-offset", limit: "test-limit",
```
Comment (wmews-hw) [inline test/Hyperwallet.spec.js:3146]: shouldn't it be formatted in many lines same as the other?

File: test/Hyperwallet.spec.js [hunk 4]
```diff
@@ -3141,7 +3218,9 @@ describe("Hyperwallet", () => {
         /** @test {Hyperwallet#listBalancesForAccount} */
         it("should do get call with options", () => {
             const callback = () => null;
-            const options = { currency: "test-currency", sortBy: "test-sortBy", offset: "test-offset", limit: "test-limit" };
+            const options = {
+                currency: "test-currency", sortBy: "test-sortBy", offset: "test-offset", limit: "test-limit",
```
Comment (wmews-hw) [inline test/Hyperwallet.spec.js:3222]: shouldn't it be formatted in many lines same as the other?

File: test/Hyperwallet.spec.js (hunk 1 above)
Comment (skoong) [reply test/Hyperwallet.spec.js:2359]: Hmm, I think it only moves it if they're not all on the same line
```
ESLint: Object properties must go on a new line if they aren't all on the same line.(object-property-newline)
```

**Pull request merged**